clouddrift/select.py

```diff
@@ -1,40 +1,41 @@
-import awkward as ak
+import xarray as xr
 import numpy as np
+import warnings
 
 
-def mask_var(var: ak.Array, value):
+def mask_var(var: xr.DataArray, value):
     """Return the mask of a subset of the data matching a test criterion.
 
     Args:
-        var (ak.Array): ak.Array
+        var (xr.DataArray): ak.Array
         value: tuple, list or scalar defining a test criterion
 
     Returns:
-        mask (Ak.Array): where values corresponding to the criterion are True
+        mask (xr.DataArray): where values corresponding to the criterion are True
```

Comment:
Fix docstring format to NumPy style, and the description should explain how different condition types work.